OpenRA.Mods.Common/Traits/Render/WithDecoration.cs

```diff
@@ -42,6 +45,9 @@ public class WithDecorationInfo : UpgradableTraitInfo, ITraitInfo
 		[Desc("Should this be visible to enemy players?")]
 		public readonly bool ShowToEnemies = false;
 
+		[Desc("Should this only be rendered when actor is in world?")]
+		public readonly bool OnlyWhenInWorld = true;
```

Comment:
What is the usecase for this?  We generally don't update the actors world position when they are inside transports etc, so you will probably end up with orphaned chevrons in weird places.